it/controllers/employment/MultipleEmploymentSummaryEOYControllerISpec.scala

```diff
@@ -92,8 +94,10 @@ class MultipleEmploymentSummaryEOYControllerISpec extends IntegrationTest with V
     val doYouNeedAnother: String = "Do you need to add another employment?"
     val yesText: String = "Yes"
     val noText: String = "No"
-    val change: String = "Change"
-    val remove: String = "Remove"
+    def change(employerName: String): String = s"Change Change employment information for $employerName"
+    def remove(employerName: String): String = s"Remove Remove employment information for $employerName"
```

Comment:
Change and remove seem to be there twice which looks odd

Reply:
It's because the selector is for the link, text and hidden text. The first change is for the text on the link and the second change part of the hidden text. This is the same with the removes

Reply:
same here